### ISSUE TITLE
Etcher 1.5.18 not running properly under Tumbleweed

### ISSUE THREAD

**Issue by fleamour** [balena-io/etcher#2715]
Opens with blank box, showing header/footer only & settings, etc, not respond to mouse clicks:

https://imgur.com/Ms9rSRe

```

Uncaught Error: /tmp/.org.chromium.Chromium.AvGBqQ: invalid ELF header
    at process.module.(anonymous function) [as dlopen] (ELECTRON_ASAR.js:166:20)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:740)
    at Object.module.(anonymous function) [as .node] (ELECTRON_ASAR.js:180:18)
    at Module.load (internal/modules/cjs/loader.js:620)
    at tryModuleLoad (internal/modules/cjs/loader.js:559)
    at Function.Module._load (internal/modules/cjs/loader.js:551)
    at Module.require (internal/modules/cjs/loader.js:658)
    at require (internal/modules/cjs/helpers.js:20)
    at Object.<anonymous> (/tmp/.mount_balenavNY3eP/resources/app.asar/node_modules/xxhash/lib/xxhash.js:4)
    at Object.<anonymous> (/tmp/.mount_balenavNY3eP/resources/app.asar/node_modules/xxhash/lib/xxhash.js:52)
    at Module._compile (internal/modules/cjs/loader.js:711)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:722)
    at Module.load (internal/modules/cjs/loader.js:620)
    at tryModuleLoad (internal/modules/cjs/loader.js:559)
    at Function.Module._load (internal/modules/cjs/loader.js:551)
    at Module.require (internal/modules/cjs/loader.js:658)
```

**Comment by lurch** [comment]
See also https://github.com/balena-io/etcher/issues/2712#issuecomment-477288405

**Comment by zvin** [comment]
Fixed in 1.5.19
